wherehows-web/app/templates/components/search-bar-form.hbs

```diff
@@ -3,7 +3,7 @@
     <label for="search-input" class="sr-only">Search</label>
 
     {{aupac-typeahead
-      action=(action (mut search))
+      action=(action "selectedSuggestion")
```

Comment:
you can use closure actions, I think that is the recommended way now.

Reply:
This is a closure action: `action=(action 'actionName')` @igbopie 
using the action helper this way creates a closure function, so this will be the lexical scope of `actionName`.
see: http://miguelcamba.com/blog/2016/01/24/ember-closure-actions-in-depth/